price-feeder/config/config_test.go

```diff
@@ -131,7 +131,7 @@ func TestValidate(t *testing.T) {
 }
 
 func TestParseConfig_Valid(t *testing.T) {
-	tmpFile, err := ioutil.TempFile("", "price-feeder.toml")
+	tmpFile, err := ioutil.TempFile("", "price-feeder*.toml")
```

Comment:
Why was this changed to `price-feeder*.toml`?

Reply:
ioutil.TempFile was appending a random string to the file name (ex: price-feeder.toml123457). This was making viper throw an error for an invalid file extension. The * moves the random string to where its set (ex: price-feeder123457.toml in this case) and away from the file extension.